client/src/components/blog/blog.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './blog.css';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import * as blogActions from '../../store/actions/blog'
+import Helper from '../../helpers/helper'
+import Loader from 'react-loader'
+
+class Blog extends Component {
+
+  static propTypes = {
+    getBlog: PropTypes.func.isRequired,
+    blog: PropTypes.object.isRequired
+  }
+
+  static defaultProps = {
+    blog: {}
+  }
+
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    this.props.getBlog(id);
+  }
+
+  delete(id) {
+    this.props.deleteBlog(id, () => {
+      this.props.history.push('/blogs');
+    });
+  }
+
+  render() {
+    return (
+      <Loader loaded={this.props.blog}>
+        <div>
+          <h2> {this.props.blog.title}</h2>
+          <div className="textCenter">
+            <h3>Written by {this.props.blog.author} at {Helper.GetLocalDateString(this.props.blog.date)}</h3>
+            <p>{this.props.blog.body}</p>
+            {this.props.blog.author === this.props.username &&
```

Comment:
export into variable `this.props.blog.author === this.props.username`, please

Reply:
done